.github/github-bot.yml

```diff
@@ -3,13 +3,6 @@ _extends: probot-settings
 github-team:
   slug: 'go'
 
-prchecklist:
```

Comment:
A friendly member from the Holy Orders of the Emperor's Inquisition will be in touch.